plugins/hls-eval-plugin/test/Main.hs

```diff
@@ -69,40 +70,43 @@ tests =
   , goldenWithEval "Refresh a multiline evaluation" "T7" "hs"
   , testCase "Semantic and Lexical errors are reported" $ do
       evalInFile "T8.hs" "-- >>> noFunctionWithThisName" "-- Variable not in scope: noFunctionWithThisName"
-      evalInFile "T8.hs" "-- >>> \"a\" + \"bc\"" $
-        if ghcVersion == GHC90
-          then "-- No instance for (Num String) arising from a use of ‘+’"
-          else "-- No instance for (Num [Char]) arising from a use of ‘+’"
+      evalInFile "T8.hs" "-- >>> res = \"a\" + \"bc\"" $
+        if
+           | ghcVersion == GHC92 -> "-- No instance for (Num String) arising from a use of `+'\n-- In the expression: \"a\" + \"bc\"\n-- In an equation for `res': res = \"a\" + \"bc\""
+           | ghcVersion == GHC90 -> "-- No instance for (Num String) arising from a use of ‘+’"
+           | otherwise -> "-- No instance for (Num [Char]) arising from a use of ‘+’"
       evalInFile "T8.hs" "-- >>> \"" "-- lexical error in string/character literal at end of input"
       evalInFile "T8.hs" "-- >>> 3 `div` 0" "-- divide by zero"
   , goldenWithEval "Applies file LANGUAGE extensions" "T9" "hs"
-  , goldenWithEval "Evaluate a type with :kind!" "T10" "hs"
-  , goldenWithEval "Reports an error for an incorrect type with :kind!" "T11" "hs"
-  , goldenWithEval "Shows a kind with :kind" "T12" "hs"
-  , goldenWithEval "Reports an error for an incorrect type with :kind" "T13" "hs"
+  , goldenWithEval' "Evaluate a type with :kind!" "T10" "hs" (if ghcVersion == GHC92 then "ghc92.expected" else "expected")
```

Comment:
This seems a bit annoying but I don't know this file well enough to have any better ideas :shrug:

Reply:
Really, all I wanted to ask was: do *you* have any other ideas? If you had one and it seemed nice and you just hadn't got around to it then I might suggest we do it here. But it sounds like you thought about it and this is the best thing for now, so I'm happy with that!